src/nr_openai_observability/bedrock.py

```diff
@@ -9,64 +8,83 @@
 from datetime import datetime
 from nr_openai_observability.monitor import monitor
 from nr_openai_observability.patcher import _patched_call
-from nr_openai_observability.consts import MessageEventName, SummaryEventName, TransactionBeginEventName
+from nr_openai_observability.consts import (
+    MessageEventName,
+    SummaryEventName,
+    TransactionBeginEventName,
+)
 from nr_openai_observability.error_handling_decorator import handle_errors
 
 
 logger = logging.getLogger("nr_openai_observability")
 logger.addHandler(logging.StreamHandler(sys.stdout))
 logger.setLevel(logging.INFO)
 
+
 def perform_patch_bedrock():
     from newrelic.common.package_version_utils import get_package_version_tuple
 
-    (major, minor, revision) = get_package_version_tuple('botocore')
+    botocore_version = get_package_version_tuple("botocore")
```

Comment:
If the library is not present, this seems to return None (which didn't like the unpacking syntax)

Reply:
Oh, neat :)